### ISSUE TITLE
Survival Crafting

### ISSUE THREAD

**Issue by dags-** [Conquest-Reforged/ReforgedMod#10]
Still not clear what the intention is for this.

I keep hearing that recipes won't be a thing, but then nothing tangeable  to use in place?

**Comment by HIPPOKLEIDES** [comment]
I think when "recipes won't be a thing" is said, it is intended to mean that they won't be working/fully functional in the initial 1.15/creative release. 

The idea for recipes is to use and extend the workstation system Mojang implemented in 1.14 with the stone-cutter. Recipes can now be designated as only working with certain stations, so we'd extend this system by adding additional workstations that handle other materials or types of blocks, like a carpentry table. We could further extend this system by making it so that the workstation can take two or more block inputs rather than just one. This would be different from regular, "crafting-table" crafting because the workstations can have multiple potential outputs, rather than just one. 

Not only would this be easier for us, as we don't have to worry about recipes fitting into a 3x3 table or conflicting with one another, but also it'd be easier for players, as they wouldn't have thousands of unique recipes to remember. 

tldr
-recipes for shape variants and certain blocks would take advantage of the new workstation/stone-cutter mechanic 
-new workstations would be added for various materials and block types
-new workstations could also take multiple inputs, but spit out multiple outputs unlike the crafting-table
-given amount of blocks we have, easier for us and players

**Comment by CreativeRealms** [comment]
Closed until 1.18.
